api/doc/spec.yaml

```diff
@@ -9486,6 +9487,9 @@ components:
           type: boolean
           description: Whether the function can be called as an entry function directly
             in a transaction
+        is_view_function:
```

Comment:
nit: Why not just is_view. This would be more consistent with is_entry

Reply:
Good call